apps/fluent-tester/src/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -1,6 +1,10 @@
 import * as React from 'react';
-import { Notification } from '@fluentui-react-native/notification';
+import { Notification, NotificationVariant, NotificationVariants } from '@fluentui-react-native/notification';
 import { Test, TestSection, PlatformStatus } from '../Test';
+import { Animated, Button, Easing, StyleSheet, Switch, TextInput, View } from 'react-native';
```

Comment:
I wouuuuld not use "Button" from React Native. That control is pretty old. I would prefer we use our own Button here.